.github/workflows/release-master.yaml

```diff
@@ -72,6 +72,24 @@ jobs:
           path: /tmp/digests/*
           if-no-files-found: error
           retention-days: 1
+  helm:
+    permissions:
+      contents: read
+      packages: write
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GH_PAT }}
+      
+      - name: Build and Push the Helm Charts to GitHub Container Registry
+        uses: JimCronqvist/action-helm-chart-repo@master
```

Comment:
how does this action know where is the chart?

And how does it generate helm version?

Reply:
My mistake,

we would need to add the following: (meaning specify the chart-path, I think default behavior is to look at root)
```YAML
    - name: Build and Push the Helm Charts to GitHub Container Registry
        uses: JimCronqvist/action-helm-chart-repo@master
        chart-path: deploy/charts/kmesh-helm
```